### PR TITLE
Judaism 2.0

### PR DESCRIPTION
**Pull request by HarrierBernstein76**
## About The Pull Request
Re-release of my Functional Judaism PR.  This time with a working circumcision knife and sprite.

## Why It's Good For The Game
It is good for inclusivity, realism, and Semitic representation within the f13 community.

### CODE REVIEW

**Comment by xTrainx** [comment]
I though we already had a satanic bible for the preacher?

**Comment by HarrierBernstein76** [comment]
> I though we already had a satanic bible for the preacher?

Please leave your anti-semitism at the door.  This is a serious PR.

File: code/modules/judaism/rabbi.dm [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	uniform =  /obj/item/clothing/under/lawyer/black/rabbi
+	shoes = /obj/item/clothing/shoes/laceup
+	suit = /obj/item/clothing/suit/toggle/lawyer/black/rabbi
+	head = /obj/item/clothing/head/pp/jhat
+	neck = /obj/item/clothing/neck/necklace/dope/rabbi
+
+/obj/item/book/manual/talmud
+	name = "The Talmud"
+	icon_state ="book1"
+	author = "Satan"
```
Comment (Kitsunemitsu) [inline code/modules/judaism/rabbi.dm:89]: "Leave your Anti-Semitism at the door"
https://tenor.com/view/senator-armstrong-armstrong-metal-gear-rising-mgr-speech-bubble-gif-25397532

Comment (HarrierBernstein76) [reply code/modules/judaism/rabbi.dm:89]: Yes, and?

**Comment by OliOliOnsiPree** [comment]
PP Station 13

**Comment by githubuser4141** [comment]
Why the fixation on circumcision? The fact that all your PRs have been centered around it, you named your branch "foreskin-2.0" and that's where your vision of Judaism begins and ends makes me think it's a low effort troll post and you don't really care about "inclusivity and Semitic representation" as your PR states.

**Pull request closed without merging**